service/resources/server_resource.go

```diff
@@ -601,6 +602,7 @@ func createAppNameList(obj interface{}) []interface{} {
 // QSLHandler handles requests for QSL
 func (s *ServerResource) QSLHandler(w http.ResponseWriter, r *http.Request) { //
 	queryMap := r.URL.Query()
+	w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
looks good, thanks as well for adding it at head of fn before much else happens, if it's set after potential error pathways, there can be an odd scenario where successful responses from route have cors header but error responses ALSO cause cors errors

in other handlers I added a bit of commentary about that, add here if you think helpful but I don't believe it's reqd